integrations/huggingface-transformers/recipes/noprune10epoch.yaml

```diff
@@ -2,5 +2,5 @@ version: 1.1.0
 
 modifiers:
     - !EpochRangeModifier
-        end_epoch: 10.0
+        end_epoch: 30.0
```

Comment:
should this file now be changed to "noprune30epochs.yaml"?

Reply:
Done